### PR TITLE
Improve documentation 1 to implement 1

### LINKED ISSUE #32: The anchors are not defined well
**Describe the feature or change you'd like**
Currently the anchors defined in client do not meet the requirements of snek-tonic. Therefore, the package imports should be revised throughout the code.

**Additional context**

### LINKED ISSUE #33: The code should be adapted to the new coding guideline
**Describe the feature or change you'd like**
Due to the change in the coding guideline on Friday, 8. May 2020 the code should be adjusted over time. 

**Additional context**
Ref: snek-at/tonic/pull/1

### PR DESCRIPTION
**Pull request by schettn**
**Checklist:**

 
 - [x] Have you added an explanation of what your changes do and why you'd like them to be included?
 - [x] Have you updated or added documentation for the change?
 - [x] Have you tested your changes with successful results?

**Type of Changes:**

 
 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [ ] New feature (non-breaking change which adds functionality)
 - [x] Documentation (non-breaking change which adds documentation)
 - [ ] Breaking change (fix or feature that would cause existing functionality to change)

**What is the current behavior? (link to any open issues here)**
- Currently the anchor definitions do not follow our standard. #32 
- Currently the documentation does not follow our standard. #33

**What is the new behavior (if this is a feature change)?**
- Now the anchor definitions do follow our standard. Closes #32 
- Now the documentation does follow our standard. Closes #33

**Other information**:
- Ref: https://githb.com/snek-at/tonic/
- Ref: 🐍

### CODE REVIEW

File: .vscode/settings.json
Comment (Aichnerc) [inline .vscode/settings.json]:
```suggestion
     "_comment": "Set the default",
```

File: .vscode/settings.json
Comment (Aichnerc) [inline .vscode/settings.json]:
```suggestion
    "_comment": "Enable per-language",
```

File: src/templates/snek/gql/tasks/general.ts
Comment (Aichnerc) [inline src/templates/snek/gql/tasks/general.ts]: Is this indentation on point?

File: src/templates/snek/gql/tasks/general.ts
Comment (Aichnerc) [inline src/templates/snek/gql/tasks/general.ts]: Is this indentation on point?

File: src/templates/snek/gql/tasks/index.ts
Comment (Aichnerc) [inline src/templates/snek/gql/tasks/index.ts]: Is this indentation on point?

**Review by Aichnerc — CHANGES_REQUESTED** [review]
🥐 As crooked as this 𝖈𝖗𝖔𝖎𝖘𝖘𝖆𝖓𝖙.

File: .vscode/settings.json
Comment (schettn) [reply .vscode/settings.json]: This is not needed here. But it is a good idea for default json files. Thx.

File: src/endpoints/index.ts
Comment (pinterid) [inline src/endpoints/index.ts]: Change "nmp" to "npm"

File: src/endpoints/scraper.ts
Comment (pinterid) [inline src/endpoints/scraper.ts]: Missing dot at the end of line.

File: src/session/index.ts
Comment (pinterid) [inline src/session/index.ts]: Change "nmp" to "npm"

File: src/session/sessions.ts
Comment (pinterid) [inline src/session/sessions.ts]: Change "nmp" to "npm"

File: src/session/sessions.ts
Comment (pinterid) [inline src/session/sessions.ts]: Change "nmp" to "npm"

File: src/session/sessions.ts
Comment (pinterid) [inline src/session/sessions.ts]: Remove dot.

File: src/session/sessions.ts
Comment (pinterid) [inline src/session/sessions.ts]: Remove dot.

File: src/session/sessions.ts
Comment (pinterid) [inline src/session/sessions.ts]: Remove dot,

File: src/templates/snek/gql/mutations/jwtAuth.ts
Comment (pinterid) [inline src/templates/snek/gql/mutations/jwtAuth.ts]: Change "nmp" to "npm"

File: src/templates/snek/gql/mutations/user.ts
Comment (pinterid) [inline src/templates/snek/gql/mutations/user.ts]: Change "nmp" to "npm"

File: src/templates/snek/gql/queries/general.ts
Comment (pinterid) [inline src/templates/snek/gql/queries/general.ts]: Change "nmp" to "npm"

File: src/templates/snek/gql/queries/user.ts
Comment (pinterid) [inline src/templates/snek/gql/queries/user.ts]: Change "nmp" to "npm" and remove one apostrophe.

File: src/templates/snek/gql/queries/user.ts
Comment (pinterid) [inline src/templates/snek/gql/queries/user.ts]: Change ":" to "."

File: src/templates/snek/gql/tasks/user.ts
Comment (pinterid) [inline src/templates/snek/gql/tasks/user.ts]: Missing dot at the end of line.

File: src/templates/snek/gql/tasks/user.ts
Comment (pinterid) [inline src/templates/snek/gql/tasks/user.ts:131]: Missing newline.

**Review by pinterid — CHANGES_REQUESTED** [review]
Looks good after some changes.

**Comment by pinterid** [comment]
The type of change has to be set before merging!

File: src/templates/snek/gql/mutations/jwtAuth.ts
Comment (pinterid) [inline src/templates/snek/gql/mutations/jwtAuth.ts]: There is on " to much.

File: src/templates/snek/gql/mutations/user.ts
Comment (pinterid) [inline src/templates/snek/gql/mutations/user.ts]: There is one " to much.

File: src/templates/snek/gql/queries/general.ts
Comment (pinterid) [inline src/templates/snek/gql/queries/general.ts]: There is on " to much.

File: src/templates/snek/gql/queries/user.ts
Comment (pinterid) [inline src/templates/snek/gql/queries/user.ts]: There is one " to much.

**Review by pinterid — CHANGES_REQUESTED** [review]
Looks good after some changes.

**Review by pinterid — APPROVED** [review]
Looks good to merge.

**Review by Aichnerc — APPROVED** [review]
LGTM 🥞

File: src/endpoints/apollo.ts
Comment (kleberbaum) [inline src/endpoints/apollo.ts]: Please remove the period.

File: src/endpoints/index.ts
Comment (kleberbaum) [inline src/endpoints/index.ts]: Please remove the period.

File: src/index.ts
Comment (kleberbaum) [inline src/index.ts:23]: Please fix indentation.

File: src/index.ts
Comment (kleberbaum) [inline src/index.ts]: and add documentation

File: src/session/sessions.ts
Comment (kleberbaum) [inline src/session/sessions.ts:69]: Please add a newline.

File: src/templates/snek/gql/tasks/auth.ts
Comment (kleberbaum) [inline src/templates/snek/gql/tasks/auth.ts]: Please fix indentation.

**Review by kleberbaum — APPROVED** [review]
Should be ok to merge after a few changes.

**Review by kleberbaum — APPROVED** [review]
Should be ok to merge.

**Pull request merged**
